dvc/repo/init.py

```diff
@@ -84,6 +84,10 @@ def init(root_dir=os.curdir, no_scm=False, force=False, subdir=False):
 
     if scm.ignore_file:
         scm.add([os.path.join(dvc_dir, scm.ignore_file)])
-        logger.info("\nYou can now commit the changes to git.\n")
```

Comment:
This will break `dvc init --no-scm` case. Let's print this message as-is and put the "initialized" message before this `if`.

Reply:
ping @Njuhobby

Reply:
Nice. Didn't realize dvc init could run in no-scm environment. Was wondering why the message was put under the if statement.